tests/PostTest.php

```diff
@@ -176,4 +176,19 @@ public function testPostRelationConnections()
 
         $this->assertEquals('no_prefix', $post->author->getConnectionName());
     }
+
+    public function testMenuItemActsAsObjectFacadeForTitleAndSlug()
+    {
+        $post = Post::find(16);
+        $this->assertEquals('Sample Page', $post->title);
+        $this->assertEquals('sample-page', $post->slug);
+
+        $post = Post::find(17);
+        $this->assertEquals('Hello world!', $post->title);
+        $this->assertEquals('hello-world', $post->slug);
+
+        $post = Post::find(18);
```

Comment:
Please, check these posts ID. I think this records exist only inside your database, not the base one in `tests/config/database.sql`.